app/locales/en.json

```diff
@@ -9,112 +9,115 @@
     "done": "Done"
   },
   "history": {
-    "no_exposure": "No known",
-    "possible_exposure": "Possible",
+    "coming_soon": "Coming Soon",
```

Comment:
You will need to make these same changes in `ht.json` because these are not managed by the trans management system.